inspirehep/config.py

```diff
@@ -358,63 +358,21 @@
         'application/json': 'invenio_records_rest.serializers:json_v1_response',
         'application/vnd+inspire.record.ui+json': INSPIRE_SERIALIZERS + ':json_literature_ui_v1_response',
         'application/x-bibtex': INSPIRE_SERIALIZERS + ':bibtex_v1_response',
-        'application/x-latexeu': INSPIRE_SERIALIZERS + ':latexeu_v1_response',
-        'application/x-latexus': INSPIRE_SERIALIZERS + ':latexus_v1_response',
-        'application/x-cvformatlatex': INSPIRE_SERIALIZERS + ':cvformatlatex_v1_response',
-        'application/x-cvformathtml': INSPIRE_SERIALIZERS + ':cvformathtml_v1_response',
-        'application/x-cvformattext': INSPIRE_SERIALIZERS + ':cvformattext_v1_response',
-        'application/x-impact.graph+json': INSPIRE_SERIALIZERS + ':impactgraph_v1_response',
         'application/marcxml+xml': INSPIRE_SERIALIZERS + ':marcxml_v1_response',
     },
     'search_serializers': {
         'application/json': 'invenio_records_rest.serializers:json_v1_search',
         'application/vnd+inspire.record.ui+json': INSPIRE_SERIALIZERS + ':json_literature_ui_v1_search',
         'application/x-bibtex': INSPIRE_SERIALIZERS + ':bibtex_v1_search',
-        'application/x-latexeu': INSPIRE_SERIALIZERS + ':latexeu_v1_search',
-        'application/x-latexus': INSPIRE_SERIALIZERS + ':latexus_v1_search',
-        'application/x-cvformatlatex': INSPIRE_SERIALIZERS + ':cvformatlatex_v1_search',
-        'application/x-cvformathtml': INSPIRE_SERIALIZERS + ':cvformathtml_v1_search',
-        'application/x-cvformattext': INSPIRE_SERIALIZERS + ':cvformattext_v1_search',
-        'application/vnd+inspire.ids+json': 'inspirehep.modules.api.v1.common_serializers:json_recids_response',
```

Comment:
This was used for something I don't really remember now. Maybe the editor @harunurhan? or maybe to do second-order searches like `refersto` and `citedby`?

Reply:
Looks like it was added in #2840 for the record-editor.

Reply:
yeah it was for editor search.

How it works in the editor is that you search and navigate search results one by one. Since it can only show single record at a time and it needs db records rather than ES, it doesn't need whole (up to 20) ES records.